medusa/providers/torrent/json/btn.py

```diff
@@ -252,17 +275,17 @@ def _api_call(self, params=None, results_per_page=300, offset=0):
             )
             time.sleep(cpu_presets[app.CPU_PRESET])
         except jsonrpclib.jsonrpc.ProtocolError as error:
-            message = error.args[0]
-            if message == (-32001, 'Invalid API Key'):
+            code, message = normalize_protocol_error(error)
+            if (code, message) == (-32001, 'Invalid API Key'):
                 log.warning('Incorrect authentication credentials.')
-            elif message == (-32002, 'Call Limit Exceeded'):
+            elif (code, message) == (-32002, 'Call Limit Exceeded'):
                 log.warning('You have exceeded the limit of 150 calls per hour.')
-            elif isinstance(message, tuple) and message[1] in (500, 524, ):
+            elif code in (500, 524, ):
```

Comment:
I dont think having the space after the comma is common practice. Just as well might remove that. Or just use `(500, 524)`